stores/sql/mysql/migrations/main/schema.sql

```diff
@@ -222,15 +221,16 @@ CREATE TABLE `host_allowlist_entry_hosts` (
   CONSTRAINT `fk_host_allowlist_entry_hosts_db_host` FOREIGN KEY (`db_host_id`) REFERENCES `hosts` (`id`) ON DELETE CASCADE
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
 
--- dbHostAnnouncement
-CREATE TABLE `host_announcements` (
+-- host_addresses contains addresses that the host announced itself with
+CREATE TABLE `host_addresses` (
```

Comment:
missing migrations?

Reply:
I'll add one in the base branch once everything is ready. `its-happening`